src/test/scala/org/clulab/habitus/variables/TestPrinting.scala

```diff
@@ -10,7 +10,8 @@ import java.io.File
 class TestPrinting extends Test {
   implicit val formats: DefaultFormats.type = org.json4s.DefaultFormats
 
-  val inputDir = "./src/test/resources"
+  val inputDir = "./src/test/resources/txt"
```

Comment:
For testing I think it's perfectly fine to rely on the development environment being present.